tokens/src/core/components/Avatar.json

```diff
@@ -1,16 +1,16 @@
 {
   "size": {
     "avatar": {
-      "base": { "value": "3rem", "type": "dimension", "source": "$avatar-size" },
-      "xs": { "value": "1.5rem", "type": "dimension", "source": "$avatar-size-xs" },
-      "sm": { "value": "2.25rem", "type": "dimension", "source": "$avatar-size-sm" },
-      "lg": { "value": "4rem", "type": "dimension", "source": "$avatar-size-lg" },
-      "xl": { "value": "6rem", "type": "dimension", "source": "$avatar-size-xl" },
-      "xxl": { "value": "11.5rem", "type": "dimension", "source": "$avatar-size-xxl" },
-      "huge": { "value": "18.75rem", "type": "dimension", "source": "$avatar-size-huge" },
+      "base": { "source": "$avatar-size", "$value": "3rem", "$type": "dimension" },
+      "xs": { "source": "$avatar-size-xs", "$value": "1.5rem", "$type": "dimension" },
+      "sm": { "source": "$avatar-size-sm", "$value": "2.25rem", "$type": "dimension" },
+      "lg": { "source": "$avatar-size-lg", "$value": "4rem", "$type": "dimension" },
+      "xl": { "source": "$avatar-size-xl", "$value": "6rem", "$type": "dimension" },
+      "xxl": { "source": "$avatar-size-xxl", "$value": "11.5rem", "$type": "dimension" },
+      "huge": { "source": "$avatar-size-huge", "$value": "18.75rem", "$type": "dimension" },
```

Comment:
super nit: might recommend keeping `source` the last property defined, as the other properties like `$value`, and `$type` are more critical.

Reply:
Agreed, that's a good idea 👍 
I would also leave some documentation somewhere about what rules to follow when creating/editing design tokens. Since it's not entirely obvious what sequence should be present in design tokens. I think we should stop at something like this:
1. `$value`
2. `description`
3. `modify`
4. `source`

Reply:
[inform]: We have a draft PR providing additions of a linter for JSON files and a sorter script
https://github.com/openedx/paragon/pull/3285